Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -2721,6 +2742,29 @@ bool ComputedStyleExtractor::hasProperty(CSSPropertyID propertyID)
     return propertyValue(propertyID);
 }
 
+static bool rendererHasTrimmedMargin(const RenderElement* renderer, CSSPropertyID propertyId)
+{
+    auto* box = dynamicDowncast<RenderBox>(renderer);
+    if (!box)
+        return false;
+
+    switch (propertyId) {
+    case CSSPropertyMarginTop:
+        return box->isFlexItem() && box->hasTrimmedMargin(PhysicalDirection::Top);
+    case CSSPropertyMarginRight:
+        return box->isFlexItem() && box->hasTrimmedMargin(PhysicalDirection::Right);
+    case CSSPropertyMarginLeft:
+
```

Comment:
/* Unofficial Review */
Intended fallthrough? If so, this new line is redundant.